dspace-api/src/test/java/org/dspace/app/bulkedit/MetadataImportTest.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.bulkedit;
+
+import static junit.framework.TestCase.assertTrue;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.AbstractIntegrationTest;
+import org.dspace.app.launcher.ScriptLauncher;
+import org.dspace.app.scripts.handler.impl.TestDSpaceRunnableHandler;
+import org.dspace.content.Community;
+import org.dspace.content.Item;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.CollectionService;
+import org.dspace.content.service.CommunityService;
+import org.dspace.content.service.ItemService;
+import org.dspace.services.ConfigurationService;
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.junit.Test;
+
+public class MetadataImportTest extends AbstractIntegrationTest {
+
+    private ItemService itemService = ContentServiceFactory.getInstance().getItemService();
+    private CollectionService collectionService = ContentServiceFactory.getInstance().getCollectionService();
+    private CommunityService communityService = ContentServiceFactory.getInstance().getCommunityService();
+    private ConfigurationService configurationService = DSpaceServicesFactory.getInstance().getConfigurationService();
+
+    @Test
+    public void metadataImportTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Community community = communityService.create(null, context);
+        collectionService.create(context, community);
+
+        String fileLocation = configurationService.getProperty("dspace.dir") + testProps.get("test.importcsv")
+                                                                                        .toString();
+        String[] args = new String[] {"metadata-import", "-f", fileLocation, "-e", eperson.getEmail(), "-s"};
+        TestDSpaceRunnableHandler testDSpaceRunnableHandler = new TestDSpaceRunnableHandler();
+
+        ScriptLauncher.handleScript(args, ScriptLauncher.getConfig(kernelImpl), testDSpaceRunnableHandler, kernelImpl);
+        Item importedItem = itemService.findAll(context).next();
```

Comment:
I understand why it can seem fragile, but as every test is cleaning up after itself there is only a single item in the iterator. There is no other way to retrieve the item that was ingested, as the handleScript() method doesn't provide output that we can parse to do a specific lookup for an item.